### PR TITLE
Feature/shoal processing

### PR DESCRIPTION
**Pull request by ruxandra-valcu**

### CODE REVIEW

**Comment by ruxandra-valcu** [comment]
Don't merge/review this yet, there's still a fix I need to put in on Monday

File: oceanstream/L3_regridded_data/shoal_process.py [hunk 1]
```diff
@@ -0,0 +1,190 @@
...
+        return None
+    filename = md.source_filenames.values
+    label = mc.attrs["label"]
+
+    ping_true = md["Sv"].any(dim="range_sample")
+    range_true = md["Sv"].any(dim="ping_time")
+    subset_md = md.sel(ping_time=ping_true, range_sample=range_true)
+
+    # TODO figure out exactly how the shiny tool does plotting
+    # bbox_0 = subset_md.ping_time.min().values
```
Comment (beatfactor) [inline oceanstream/L3_regridded_data/shoal_process.py]: I think the bounding boxes should be:
```py
bbox_0 = float(subset_md.ping_time.min())
bbox_1 = float(subset_md.range_sample.min())
bbox_2 = float(subset_md.ping_time.max())
bbox_3 = float(subset_md.range_sample.max())
```

**Comment by beatfactor** [comment]
@mihaiboldeanu can you review please?

File: oceanstream/L3_regridded_data/shoal_process.py (hunk 1 above)
Comment (ruxandra-valcu) [reply oceanstream/L3_regridded_data/shoal_process.py]: So did I, at first, but this is not what the toolset consuming these is using. This portion can be ignored for now, I'll fix it in a further commit today

**Review by mihaiboldeanu — APPROVED** [review]
Looks ok!

**Review by mihaiboldeanu — APPROVED** [review]

**Pull request merged**